docs/django-admin/roles.md

```diff
@@ -9,6 +9,28 @@ our `user_group` model and run in a migration.
 
 For more details, refer to the [user group model](../../src/registrar/models/user_group.py).
 
+## Adding a user as analyst or granting full access via django-admin
+
+If a new team member has joined whose data is NOT included in fixtures,
+then they will need to be granted analyst ( `cisa_analysts_group`) or full access (`full_access_group`) permissions in order to view the django-admin pages.
+To do this, do the following:
+
+1. The user in question will need to have a login.gov account and login into our system, this will create a `Users` table entry with their email address and name.
+2. On that `Users` table note that the `GROUP` column should be blank for them as they have no special permisisons yet.
+3. Click on their username, then scroll down to the `Permissions` section.
+4. Under `Permissions`, see the `Groups` table which has a column for `Available groups` and `Chosen groups`. Select the permission you want from the `Available groups` column and click the right arrow to move it to the  `Chosen groups`. Note, if you want this user to be an analyst select `cisa_analysts_group`, otherwise select the `full_access_group`.
```

Comment:
This step could benefit from a screenshot (could just be the page they should do this on) or link to a loom just to help orient somebody who may be lost at this stage. I think its fairly intuitive but usually a visual reference for chained steps helps orient people faster. 

If a screenshot would look weird here and/or goes against style rules, splitting this up into separate steps would also help and I think would work just as well.

Reply:
Yeah unforntunetly the screen shot wouldn't work great here but agreed it would be nice to include one

Reply:
Why couldn't a screenshot be included in this Markdown file?

Reply:
@abroddrick Its not blocking at all, but I think the main thing for this is not a screenshot but that step 4 needs additional context or to be kind of "spaced out" if that makes sense. Doing so would help it be easier to follow, though its not necessarily hard to follow now (in other words, its nitpicky). For an example, I like how the instructions are split up in another section of your code, below: 
![image](https://github.com/cisagov/getgov/assets/141044360/85e18f7f-f970-4081-9e9b-05f7633ef063)